src/dvclive/dvc.py

```diff
@@ -156,3 +157,29 @@ def get_random_exp_name(scm, baseline_rev):
         exp_ref = ExpRefInfo(baseline_sha=baseline_rev, name=name)
         if not scm.get_ref(str(exp_ref)):
             return name
+
+
+def get_dvc_stage_template(live):
+    stage = {
+        "cmd": "<python my_code_file.py my_args>",
+        "deps": ["<my_code_file.py>"],
+        "outs": [],
+    }
+    rel_path = os.path.relpath(os.getcwd(), live._dvc_repo.root_dir)
```

Comment:
Can we add a test for this? I.e. using `chdir` inside a subdirectory in the test

Reply:
Added a test. PTAL.